index.js

```diff
@@ -139,9 +164,25 @@ function zeroPadNumToN(num, n) {
 	return zeroPadded;
 }
 
+// For shuffling arrays:
+// - construct a seed from the salt (just use the salt string)
+// - created a seeded prng (via knuth-shuffle-seeded, added to package.json, 
+// -- https://www.npmjs.com/package/knuth-shuffle-seeded)
+// -- https://github.com/TimothyGu/knuth-shuffle-seeded
+// - construct reversible shuffle of article id
```

Comment:
I don't think these comments are needed, the code itself explains the steps taken